test/agentchat/test_groupchat.py

```diff
@@ -514,4 +588,5 @@ def test_selection_helpers():
     # test_agent_mentions()
     # test_termination()
     # test_next_agent()
-    test_invalid_allow_repeat_speaker()
+    # test_invalid_allow_repeat_speaker()
```

Comment:
why was this commented out?

Reply:
Oh, sorry, I didn't know it's needed. Thought that everything after `if` __name__ == "__main__":` is just test poligon, and we commenting out all the functions except the last one ;) Uncommented it back.

Reply:
You are right, my mistake. Sorry about that